openweather-based-bidding/classes/config.gs

```diff
@@ -58,23 +58,38 @@ class Config {
       'sheet-name': '',
 
       // Column mappings
-      'col-line-item-id': 0,
-      'col-insertion-order-id': 1,
-      'col-advertiser-id': 2,
-      'col-lat': 3,
-      'col-lon': 4,
-      'col-temp-min': 5,
-      'col-temp-max': 6,
-      'col-precip': 7,
-      'col-wind-min': 8,
-      'col-temp-curr': 9,
-      'col-precip-curr': 10,
-      'col-wind-curr': 11,
-      'col-status': 12,
-      'col-last-update': 13
+      'col-line-item-id':       'Line Item Id',
+      'col-insertion-order-id': 'Insertion Order Id',
+      'col-advertiser-id':      'Advertiser ID',
+      'col-lat':                'Latitude',
+      'col-lon':                'Longitude',
+      'col-formula':            'Activation Formula',
+
+      // Spreadsheet headers
+      'headers': [],
     };
   }
 
+  /**
+   * Set headers array for further processing
+   *
+   * @param headers Headers array
+   */
+  setHeaders(headers) {
+    this.config.headers = headers;
+  }
+
+  /**
+   * Returns the index of the header entity
+   * 
+   * @param idx Header notation (from `this.config`) we are looking for
+   * @return {integer} Index, if not exists then -1.
+   */
+  getHeaderIndex(idx, plusOne) {
```

Comment:
`plusOne` is missing in the doc string.
I suggest calling it `zeroBased` and if `!zeroBased` -> add 1 (or let the caller deal with that entirely)

Reply:
Good point. I decided to rename it to the "indexShift" and slightly refactored the function